examples/recipes/non-additivity/queries/run.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+host=cube
+port=4000
+loadUrl=cubejs-api/v1/load
+readyzUrl=readyz
+
+token=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpYXQiOjEwMDAwMDAwMDAsImV4cCI6NTAwMDAwMDAwMH0.OHZOpOBVKr-sCwn8sbZ5UFsqI3uCs6e4omT7P6WVMFw
```

Comment:
I think we can remove `token` and send queries without him

Reply:
I see your point! However, I suggest keeping it in all `run.sh` files because skipping the token it's only possible in dev mode, but it's mandatory (=best practice) in production.